pycheribuild/projects/bluespec_compiler.py

```diff
@@ -45,11 +45,20 @@ def __init__(self, config: CheriConfig):
         self.add_required_system_tool("cabal", apt="cabal-install", homebrew="cabal-install")
         for i in ("autoconf", "gperf", "bison", "flex"):
             self.add_required_system_tool(i, homebrew=i)
+        self.add_required_system_header("tcl/tcl.h", apt="tcl-dev")
         self.make_args.set(PREFIX=self.install_dir)
 
+    def clean(self, **kwargs):
```

Comment:
It looks like latest version support out-of-source builds, if we can use that instead we can avoid changing `clean()`.